cve_bin_tool/parsers/__init__.py

```diff
@@ -15,6 +15,7 @@
     "swift",
     "php",
     "perl",
+    "rpm"
```

Comment:
```suggestion
    "rpm",
```

Pretty sure this is the remaining black issue, so I'm just going to merge a suggestion to fix it.